iceoryx_hoofs/test/moduletests/test_cxx_string.cpp

```diff
@@ -39,6 +39,15 @@ using Implementations = Types<string<1>, string<15>, string<100>, string<1000>>;
 TYPED_TEST_CASE(stringTyped_test, Implementations);
 #pragma GCC diagnostic pop
 
+TEST(string_test, CapacityReturnsSpecifiedCapacity)
+{
+    constexpr uint16_t CAPACITY_ONE{1};
+    constexpr uint16_t CAPACITY_FOURTYTWO{42};
+
+    EXPECT_THAT(string<CAPACITY_ONE>::capacity(), Eq(CAPACITY_ONE));
+    EXPECT_THAT(string<CAPACITY_FOURTYTWO>::capacity(), Eq(CAPACITY_FOURTYTWO));
+}
```

Comment:
I know this is legal, but I don't like that we call `capacity()` as if it belongs to an individual object in all the other tests. I can imagine that someone who wants to learn how the string works looks into the test file and gets the wrong impression that `capacity` is a non-static member function, e.g. when we call in l. 66
```
constexpr auto STRINGCAP = MyString().capacity();
```
instead of
```
constexpr auto STRINGCAP = MyString::capacity();
```
Here, we create a string object which is never used again only to get the capacity.

Reply:
Done